test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -44,13 +43,13 @@ public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
                 .Execute("new");
 
             new TestCommand("dotnet") { WorkingDirectory = rootPath }
-                .Execute("restore");
+                .Execute("restore3 -s https://dotnet.myget.org/F/cli-deps/api/v3/index.json");
```

Comment:
@dasMulli I discussed this with @eerhardt and we concluded that this will be a bit cleaner if we don't hard-code the source url and instead move the tests to the CLI's test directory infrastructure so they take advantage of the common NuGet.config files in the repo. I think this should address your hard-coding concern.